src/components/TransactionsInterpreter/tests/extended-transaction-mock.ts

```diff
@@ -1,13 +1,13 @@
-import { baseTransactionMock } from '../helpers/tests/base-transaction-mock';
 import {
-  TransactionOperationActionType,
-  TxActionCategoryEnum,
   TxActionsEnum,
-  UITransactionType,
-  VisibleTransactionOperationType
-} from '../helpers/types';
+  TxActionCategoryEnum,
+  VisibleTransactionOperationType,
+  TransactionOperationActionType,
+  TransactionType
+} from 'types/server-transations';
+import { baseTransactionMock } from '../helpers/tests/base-transaction-mock';
 
-export const transactionMock: UITransactionType = {
+export const transactionMock: TransactionType = {
   ...baseTransactionMock,
   // type: TransferTypeEnum.SmartContractResult,
```

Comment:
I know this is not added by you, but I appreciate it if you will remove this commented line